scripts/india/geo/districts_test.py

```diff
@@ -58,3 +58,43 @@ def test_get_district_name_to_lgd_code_mapping(self):
         district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
             "telangana", "warangal")
         self.assertEqual(district_lgd_code, "522")
+
+        district_lgd_code = mapper.get_district_name_to_lgd_code_mapping(
+            "andhra pradesh", "nandyal")
```

Comment:
Does the state lookup support queries like 'Andhrapradesh' or 'AP' ?
Should INDIA_STATES_LGD_CODES be expanded with more versions of the state names?

Reply:
@ajaits  We can add more look up formats. I will create an issue and we can add them as and when required.